src/components/simulator/Simulator.tsx

```diff
@@ -167,7 +167,7 @@ export const Simulator = ({
     let context: any = {};
 
     if (interactivePayload) {
-      type = interactivePayload.payload.type;
+      type = interactivePayload.payload.type || interactivePayload.type;
```

Comment:
Can we use sendMediaMessage function to send the location request message similar to how we are doing while using the location button in attachment list. 

Refer implementation of the location button in the attachment list menu inside simulator.

Reply:
but wouldn't there be inconsistency if we do this, because `sendMediaMessage` this function is used for all the attachment messages and this `sendMessage` for the other kind of messages (like interactive messages)

Reply:
Let's discuss this once in tomorrow's call. I think its an overlap of the two types.